src/Application/QueryStringParser.php

```diff
@@ -51,7 +51,7 @@ private function splitQueryString( string $queryString ): array {
 	}
 
 	private function isInstanceOfPart( string $part ): bool {
-		return str_starts_with( $part, 'haswbstatement:' . $this->itemTypeProperty->getSerialization() );
+		return str_starts_with( $part, 'haswbfacet:' . $this->itemTypeProperty->getSerialization() . '=' );
```

Comment:
There was a logic issue here. Valid item type "P32" was going to be a substring of non-item type "P320".